lib/Dialect/Torch/Transforms/Passes.cpp

```diff
@@ -168,12 +171,36 @@ static void createRefinementPipeline(
       mlir::torch::Torch::createDropAbstractInterpCalculationsPass());
 }
 
-void mlir::torch::Torch::createTorchShapeRefinementPipeline(OpPassManager &pm) {
+void mlir::torch::Torch::createTorchShapeRefinementPipeline(
+    OpPassManager &pm, const TorchLoweringPipelineOptions &options) {
   createRefinementPipeline(pm, Torch::createReifyShapeCalculationsPass,
-                           Torch::createSimplifyShapeCalculationsPass);
+                           Torch::createSimplifyShapeCalculationsPass, options);
 }
 
-void mlir::torch::Torch::createTorchDtypeRefinementPipeline(OpPassManager &pm) {
+void mlir::torch::Torch::createTorchDtypeRefinementPipeline(
+    OpPassManager &pm, const TorchLoweringPipelineOptions &options) {
   createRefinementPipeline(pm, Torch::createReifyDtypeCalculationsPass,
-                           Torch::createSimplifyDtypeCalculationsPass);
+                           Torch::createSimplifyDtypeCalculationsPass, options);
 }
+
+int mlir::torch::Torch::loadExtraLibrary(const std::string &extraLibraryPath,
```

Comment:
this needs to go somewhere else - let me know where.

Reply:
ReifyAbstractInterpCalculationsUtils.h would make sense.